apps/req.c

```diff
@@ -804,8 +819,9 @@ int req_main(int argc, char **argv)
                 goto end;
             if (days == UNSET_DAYS) {
                 days = DEFAULT_DAYS;
-            }
-            if (!set_cert_times(new_x509, NULL, NULL, days))
+            } else if (not_after != NULL)
+                BIO_printf(bio_err,"Warning: -not_after option overriding -days option\n");
```

Comment:
please remove `{}` around the `days = DEFAULT_DAYS;`

Reply:
Done.